README.rst

```diff
@@ -1,7 +1,10 @@
 CrateDB Docs Theme
 ==================
 
-A Sphinx_ theme for the `CrateDB documentation`_.
+A Sphinx_ theme for the `CrateDB documentation`_. This theme can be used as an 
+inspiration on how to generate and setup a Sphinx_ theme. It is not intended
```

Comment:
Maybe even make a `.. note::` block?

Reply:
Or bold?

Reply:
NOTE blocks don't work in the README

see my above-suggested edit for the emphasis